src/style.css

```diff
@@ -1 +1,30 @@
-/* add styles here */
+.page__budy {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Avoid using the '*' selector as it impacts performance. It's better to set styles only for elements that require them. Zeroing out your margins, paddings or other styles with '*' is still inefficient for the browser.